modules/command/dh/dh-find-trail-command.js

```diff
@@ -0,0 +1,96 @@
+const Command = require('../command');
+const Models = require('../../../models/index');
+
+/**
+ * Handles one data challenge
+ */
+class DHFindTrailCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.config = ctx.config;
+        this.logger = ctx.logger;
+        this.trailService = ctx.trailService;
+    }
+
+    /**
+     * Executes command and produces one or more events
+     * @param command
+     */
+    async execute(command) {
+        const {
+            handler_id,
+            unique_identifiers,
+            depth,
+            reach,
+            included_connection_types,
+            excluded_connection_types,
+        } = command.data;
+
+        const response = await this.trailService.findTrail(
+            unique_identifiers,
+            depth,
+            reach,
+            included_connection_types,
+            excluded_connection_types,
+        );
+
+        await Models.handler_ids.update(
+            {
+                data: JSON.stringify(response),
```

Comment:
This response potentially can be really big, we should not use database for storing it we could run into the same problem as we did for import. We should save it in a file.